public/app/features/alerting/unified/components/mute-timings/MuteTimingsTable.tsx

```diff
@@ -265,7 +240,79 @@ function useColumns(
       });
     }
     return columns;
-  }, [alertManagerSourceName, setMuteTimingName, showActions, exportSupported, exportAllowed, openExportDrawer]);
+  }, [
+    alertManagerSourceName,
+    setMuteTimingName,
+    showActions,
+    exportSupported,
+    exportAllowed,
+    openExportDrawer,
+    styles.actionsColumn,
+  ]);
+}
+
+function isDisabled(muteTiming: MuteTimeInterval) {
+  const allTimeIntervalsDisabled = muteTiming.time_intervals.every((timeInterval) => {
+    return (
+      timeInterval.times?.length === 0 ||
+      timeInterval.weekdays?.length === 0 ||
+      timeInterval.days_of_month?.length === 0 ||
+      timeInterval.months?.length === 0 ||
+      timeInterval.years?.length === 0
+    );
+  });
+  return allTimeIntervalsDisabled;
+}
+
+interface ActionsAndBadgeProps {
+  data: MuteTimeInterval;
```

Comment:
How about `muteTiming` instead of `data`? I had to check the `props` interface to find out what `data` is